firmware/quadruna/VC/src/tasks.c

```diff
@@ -397,6 +404,7 @@ void tasks_run100Hz(void)
 
         // Watchdog check-in must be the last function called before putting the
         // task to sleep.
+        io_telemMessage_broadcast(&modem900_uart);
```

Comment:
nit, but the standard throughout the car is that you pass this type of metadata in the init function, then store it as a static variable inside `io_telemMessage.c`